src/components/projectEditor.js

```diff
@@ -6,27 +6,61 @@ const ProjectEditor = (props) => {
 
   customerElements.push(<option disabled hidden key='-1' value='-1'></option>);
 
+  let billableElements = [{ id: true, name: 'Ja' }, { id: false, name: 'Nei' }]
```

Comment:
Perhaps `value` would be better than `id`? (Even though it's also used as an `id`.)